horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/services/SchemaService.java

```diff
@@ -239,19 +241,20 @@ int addOrUpdateTransformer(@PathParam("schemaId") int schemaId,
    @Parameters(value = {
            @Parameter(name = "id", description = "Schema ID", example = "101"),
    })
-   @APIResponseSchema(value = String.class,
-           responseDescription = "A JSON representation of the Schema object",
+   @APIResponseSchema(value = SchemaExport.class,
+           responseDescription = "A JSON representation of the SchemaExport object",
            responseCode = "200")
-   String exportSchema(@PathParam("id") int id);
+   SchemaExport exportSchema(@PathParam("id") int id);
 
    @POST
    @Path("import")
    @Consumes(MediaType.APPLICATION_JSON)
+   @APIResponse(responseCode = "204", description = "Import a new Schema or update an existing Schema")
    @RequestBody(content = @Content( mediaType = MediaType.APPLICATION_JSON,
            schema = @org.eclipse.microprofile.openapi.annotations.media.Schema(
-                   type = SchemaType.STRING, implementation = String.class)) )
-   @Operation(description="Import an previously exported Schema")
-   void importSchema(String config);
+                   type = SchemaType.STRING, implementation = SchemaExport.class)) )
```

Comment:
The OpenApi viewer is displaying the input payload as "string". This should be a JSON representation of SchemaExport

![Screenshot from 2024-02-06 06-25-32](https://github.com/Hyperfoil/Horreum/assets/959822/ef30bb42-222f-4749-a990-df2a93073c94)